src/rgw/rgw_sal_motr.h

```diff
@@ -457,7 +457,7 @@ class MotrZone : public Zone {
     virtual const RGWAccessKey& get_system_key() { return zone_params->system_key; }
     virtual const std::string& get_realm_name() { return realm->get_name(); }
     virtual const std::string& get_realm_id() { return realm->get_id(); }
-
+    virtual const std::string& get_tier_type() { return "fixme"; }
```

Comment:
I have no problem (in principle) with returning a fixed string here, but maybe it should be the default tier type (whatever that is...) rather than "fixme"?  @soumyakoduri what do you think?

Reply:
ah

Reply:
for now, I'd propose to return "rgw" rather than "fixme" :)

Reply:
yes DBZone & MotrZone are just place-holders and are not properly initialized. Returning default value "rgw" seems fine to me (along with "fixme" comment maybe) for now.